devicon.json

```diff
@@ -5776,6 +5776,36 @@
         },
         "color": "#294172", 
         "aliases": [] 
-    }
+    },
+{
```

Comment:
`{` identation 😁

Reply:
Thanks a lot really, you really helped a lot on these TwT

Reply:
Not solved yet :)